.github/workflows/release-image-builds.yml

```
@@ -162,9 +166,17 @@ jobs:
           IMAGE: ${{ matrix.image }}
           TIMESTAMP: ${{ needs.make-date-tag.outputs.dtag }}
         run: |
-          docker_repo=opensciencegrid/$IMAGE
-          tag_list=$docker_repo:$REPO,$docker_repo:$REPO-$TIMESTAMP
-          echo "::set-output name=taglist::$tag_list"
+          docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
```

```suggestion
          docker_repo=opensciencegrid/$IMAGE
```

This repo is slightly different as it builds one base image then multiple child images so we can't depend on the GitHub repo name for each of the child images